File: 2.Bolum/mantiksal_operator_2/go/main.go
package main

import "fmt"

func main() {
	var t, z int
	fmt.Scanf("%d", &t)
	fmt.Scanf("%d", &z)

	if t > 21 && z < 30 {
		fmt.Printf("Girilen sayılar 22 ile 30 arasında \n Girilen sayılar : %d , %d ", t, z)
	}

}

File: 2.Bolum/mantiksal_operator_2/c/main.c
#include <stdio.h>
#include <stdlib.h>

void main()
{
    int t, z;
    printf("iki sayi girin : ");
    scanf("%d, %d", &t, &z);
    if (t > 21 && z < 31)
        printf("Girilen sayi 22 ile 30 arasinda \n");

    puts("Bitti");
}